pkg/fakes/config.go

```diff
@@ -0,0 +1,29 @@
+package fakes
+
+import (
+	"io/ioutil"
+	"math/rand"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+func SetFakeConfig() {
+	logrus.SetOutput(ioutil.Discard)
+	viper.SetConfigName("config")
+	viper.AddConfigPath("$GOPATH/src/github.com/makerdao/vdb-transformer-utilities/pkg/fakes/")
```

Comment:
This doesn't feel right - it ties the shared library to GOPATH, which I think we're supposed to be able leave behind, and why would a shared library need configuration. You can't run it.

Reply:
This is a config file that's used to test that the config helpers (e.g. `GetContractAddress`) works. Would love to decouple it from the GOPATH and/or remove it entirely, but not sure how to achieve that - any ideas?

Reply:
Since it's a fakes dir I'm wondering if you could use a relative path here?